cmd/osctl/cmd/root.go

```diff
@@ -115,7 +117,7 @@ func setupClient(action func(*client.Client)) {
 	md.Set("targets", target...)
 	globalCtx = metadata.NewOutgoingContext(globalCtx, md)
 
-	t, creds, err := client.NewClientTargetAndCredentialsFromConfig(talosconfig)
+	t, creds, err := client.NewClientTargetAndCredentialsFromConfig(talosconfig, cmdcontext)
```

Comment:
Instead of this I think it would be better to follow the same pattern we use for `--targets` and set this in https://github.com/talos-systems/talos/blob/a4e1479b07c3951a9f4f5e38a9d5bec4eebc8549/cmd/osctl/cmd/root.go#L112

Reply:
Thank you for review! But i don't understand pattern for `--targets`. Can you explain me?

Reply:
Sorry I wasn't very clear. And looking closer at my suggest I was just wrong :)